alvr/session/src/settings.rs

```diff
@@ -22,9 +22,9 @@ pub enum FrameSize {
 #[derive(SettingsSchema, Serialize, Deserialize, Clone)]
 #[serde(rename_all = "camelCase", tag = "type", content = "content")]
 pub enum NvencPreset {
-    LowLatencyDefault = 0,
-    LowLatencyHighQuality = 1,
-    LowLatencyHighPerformance = 2,
+    Quality = 0,
+    Balanced = 1,
+    Speed = 2,
```

Comment:
Can you merge this with the equivalent option for AMF?

Reply:
Can you also expose more encoding options? like 1-pass or 2-pass encoding (i'm not sure if that's a thing, i didn't read the new header, but just to give you an idea).